docs/figma-box-shadow.md

```diff
@@ -0,0 +1,66 @@
+---
+title: "Figma Box Shadow"
+version: 0.1.0
+revision: 1
+---
+
+# Figma Box Shadow
+
+The box shadow is handled as the shadow of the effect from figma.
+
+![](https://static.figma.com/uploads/9def6cce093b164306328ee228028155d13d72d0)
+
+[figma DropShadowEffect](https://www.figma.com/plugin-docs/api/Effect/#dropshadoweffect)
+
+[W3C](https://drafts.csswg.org/css-backgrounds/#box-shadow)
+
+## drop-shadow
+
+**css**
+
+- [box-shadow](https://developer.mozilla.org/en-US/docs/Web/CSS/box-shadow)
+
+**syntax**
+
+offset-x | offset-y | color
+
+offset-x | offset-y | blur-radius | color
+
+offset-x | offset-y | blur-radius | spread-radius | color
+
+inset | offset-x | offset-y | color
+
+Any number of shadows, separated by commas
+
+```css
+box-shadow: 12px 12px 2px 1px rgba(0, 0, 255, 0.2);
+```
```

Comment:
Can be better

Reply:
Is this comment a question?
If this is a question, I'll add more info.